app/services/v1/sms/replay.rb

```diff
@@ -33,4 +46,71 @@ def delete_messaged_for(number)
       job.delete if job_args['job_class'] == 'SmsPlans::SendSmsJob' && job_args['arguments'].first.eql?(number)
     end
   end
+
+  def handle_message_with_sms_code
+    session = Session::Sms.includes(:intervention).where(intervention: { status: :draft }).find_by(sms_code: message)
+    return SmsPlans::SendSmsJob.perform_later(from_number, 'There is no such session', nil, @user&.id) unless session
```

Comment:
We support Spanish and Arabic languages, so we should adjust the message based on this.